client/src/components/ResultsTableOverall.vue

```diff
@@ -53,7 +53,10 @@
             <td scope="col" class="hide-on-sm">
               {{ flight.team?.name }}
             </td>
-            <td class="hide-on-sm">{{ flight.takeoff.name }}</td>
+            <td class="hide-on-sm">
```

Comment:
Noice :-D

Wollen wir das hier für Report, Photos und Kommentare ebenfalls ergänzen?
bi bi-camera
bi bi-card-text
bi bi-chat
Dann müsste ich dafür tatsächlich noch Flags setzen im Backend.

Reply:
Könnte man überlegen. Mal schauen wie man das alles noch optisch unterbringt aber die flags wären schon mal gut zu haben.

für report brauchen wir aber keins, der wird ja eh schon mitgeliefert in der list. (man könnte ihn aber da aus Datensparsamkeit raus nehmen)